secrets/store.go

```diff
@@ -41,6 +42,11 @@ type Store struct {
 // Context should come with a timeout otherwise this might block forever, i.e.
 // if the path never becomes available.
 func NewStore(ctx context.Context, path string, logger log.Wrapper, middlewares ...SecretMiddleware) (*Store, error) {
+	return newStoreOverrideFSEventsDelay(ctx, filewatcher.DefaultFSEventsDelay, path, logger, middlewares...)
```

Comment:
🔕 normally when a public constructor calls an internal one, I just make it the corresponding name (`newStore`), even if it has different args.  The suffixes are useful for when there is more than one of something imo